### ISSUE TITLE
[template] Should say something about how to become an invited expert

### ISSUE THREAD

**Issue by jyasskin** [w3c/charter-drafts#451]
@rhiaro's https://rhiaro.co.uk/2023/09/social-web points out that it can be hard for a non-member expert to know whether they're welcome to participate in a working group. Their article is about a potential Social Web WG, but I think its recommendation would be good to apply to WG charters in general. In particular, charters should include "A set of criteria which Invited Experts should meet" and "A clear process for individuals to self-nominate as IEs." The set of criteria should help folks realize that they're wanted even if they don't work for a W3C member.

**Comment by svgeesus** [comment]
I agree it is hard to know how to get involved, for interested members of the public.

I guess there are three aspects to becoming an IE:

1. Expert. Has the knowledge required to understand and usefully contribute to the topic
2. Invited. The chair(s) and staff contact(s) are aware of their work and want to invite them
3. Involved. Expertise is not sufficient; being involved in GitHub discussions, pull requests etc and a history of positive involvement is the biggest factor influencing point 2.

Which from the outside, sounds like a catch-22 because many people are unaware that they can become involved _without_ being an invited expert.

So the primary advice needs to be "get involved". People who productively contribute, who help in discussions, who send in PR to fix spec errors, and who do so over a span of time greater than a month or so, are much more likely to be considered. But even if they aren't, if the goal is involvement, that is open to all.

**Comment by svgeesus** [comment]
To follow up on this, it doesn't seem like something that should be in an individual charter; instead it should be in the [Guide](https://www.w3.org/Guide/). We already have

> [Join a group](https://www.w3.org/participate/faq.html#join) (see also [Invited Expert Policy](https://www.w3.org/2004/08/invexp.html))

and "join a group" links to [Get involved](https://www.w3.org/get-involved/#join) the majority of which falls under **Participate without W3C Membership**. So this aspect seems to be better documented than it was,I believe, when the issue was first raised. @jyasskin do you agree?

**Comment by jyasskin** [comment]
It needs to be in the places that a subject-matter expert (but not a W3C expert) might think to look. I don't think the Guide qualifies at all. Group home pages would be a good place for this, where they currently just have:

> To join or leave this group, please [sign in](https://auth.w3.org/?url=https%3A%2F%2Fwww.w3.org%2Fgroups%2Fwg%2Fvc%2F) to your account.
> [Patent Policy status](https://www.w3.org/groups/wg/vc/ipr/)  [How to Join](https://www.w3.org/groups/wg/vc/instructions/)

(This is bad because "sign in" implies that you should already have a W3C account, and while it links to a page that links to the "create an account" page, the "create" page says "Please create an account only if you are participating in a group at W3C.", which is the catch-22. The current text is also bad because it doesn't explain what kinds of people _should_ want to join; you have to already know that you belong.)

After the home page, a Github CONTRIBUTING file is probably the next place prospective IEs might look. I don't know that there's any central control over those?

Charters are then the third tier. I think they'd be a good place to describe and agree on any group-specific needs. Amy's "people who have stewarded communities and moderated instances in the fediverse or similar to participate, as well as current implementers, and implementers who plan to integrate AP into existing software in future" would be great to put in the Social Web WG charter. This could then get copied into the group home page and be a bit of community leverage over those home pages.

**Comment by svgeesus** [comment]
Closed by

 - https://github.com/w3c/charter-drafts/commit/d3a947391950445eb8faaf7e132a60f8b54ac78f
